exercises/practice/space-age/.meta/config.json

```diff
@@ -18,5 +18,5 @@
   },
   "blurb": "Given an age in seconds, calculate how old someone is in terms of a given planet's solar years.",
   "source": "Partially inspired by Chapter 1 in Chris Pine's online Learn to Program tutorial.",
-  "source_url": "https://pine.fm/LearnToProgram/?Chapter=01"
```

Comment:
@kytrinyx This seems to be an issue with the source. The URL should be using HTTPS and our Link checker would otherwise fail it for not using HTTPS. We had disabled it because it failed in instances of mailto links.

I need to check if that's been fixed.

Reply:
I will fix this upstream and re-sync here.